WORKSPACE

```diff
@@ -143,10 +143,10 @@ load("@bazel_gazelle//:deps.bzl", "gazelle_dependencies")
 gazelle_dependencies()
 
 # Protobuf
-http_archive(
+git_repository(
     name = "com_google_protobuf",
-    strip_prefix = "protobuf-master",
-    urls = ["https://github.com/protocolbuffers/protobuf/archive/master.zip"],
+    commit = "520c601c99012101c816b6ccc89e8d6fc28fdbb8",
+    remote = "https://github.com/protocolbuffers/protobuf",
```

Comment:
sh256 only applies to http_archive rules, not git_repository rules. git_repository rules allow specifying a commit hash (which itself is a sha hash, sha1 IIRC) which I did. Am I missing something here?

Reply:
Ah I see, since it looks like every `git_repository()`in the `WORKSPACE` file is missing a sha256, and we should probably make sure the version of bazel we are using (and retrieving in [bazelisk.sh](https://github.com/lowRISC/opentitan/blob/master/bazelisk.sh)) supports passing the sha256 field to a `git_repository()` rule, then perhaps it makes sense to address this in follow up PR(s)?

Reply:
yes